Dockerfile

```diff
@@ -20,26 +20,13 @@ ENV GO111MODULE=off
 
 FROM base AS criu
 ARG DEBIAN_FRONTEND
-# Install dependency packages specific to criu
+ADD --chmod=0644 https://download.opensuse.org/repositories/devel:/tools:/criu/Debian_11/Release.key /etc/apt/trusted.gpg.d/criu.gpg.asc
```

Comment:
May want to change the Dockerfile `syntax` to `docker/dockerfile:1` (latest stable); I think version 1.2 and older had an issue with the `--chmod` flag; see https://github.com/moby/moby/commit/882fd19f39a12124ac31cc75cf1bcd6976416a2f

Reply:
That said, we see instability with the opensuse package repositories, so we're considering switching back to compiling from source (see https://github.com/moby/moby/pull/44086)

Reply:
I have implemented the changes.